### PR TITLE
Hotpatch: Don't highlight those who use hotpatch

### PR DESCRIPTION
**Pull request by panpawn**
The client counter-part of this must be merged first: https://github.com/Zarel/Pokemon-Showdown-Client/pull/760

### CODE REVIEW

**Comment by ascriptmaster** [comment]
Update the protocol doc maybe

**Comment by panpawn** [comment]
Good idea,  I'll do that as soon as I get home! Thanks, @ascriptmaster

**Comment by panpawn** [comment]
Updated

File: PROTOCOL.md [hunk 1]
```diff
@@ -98,6 +98,10 @@ displayed inline because they happen too often. For instance, the main server
 gets around 5 joins/leaves a second, and showing that inline with chat would
 make it near-impossible to chat.
 
+Another message type is `m` followed by a timestamp from the server, followed
+by the message.  Example: `|m|timestamp|message`, this will make it so that the
+message will have a timestamp, if a user has those enabled for their settings.
+
```
Comment (Zarel) [inline PROTOCOL.md]: Um, this is just the sample section, you haven't actually updated the protocol doc...

Comment (panpawn) [inline PROTOCOL.md]: So, put it under Room Messages, yes?

Comment (Zarel) [inline PROTOCOL.md]: Put it next to `|c|` but I'm still thinking about it...

Also `|m|timestamp|message` is completely wrong; it's `|m:|timestamp|message` or `|m|message`.

Comment (panpawn) [inline PROTOCOL.md]: I'll update this as soon as I have time

**Comment by panpawn** [comment]
Updated

File: PROTOCOL.md (hunk 1 above)
Comment (panpawn) [inline PROTOCOL.md]: done!

**Pull request closed without merging**